src/components/markdown.js

```diff
@@ -1,6 +1,6 @@
 import DOMPurify from 'dompurify'
 import _ from 'lodash/fp'
-import marked from 'marked'
+import { marked } from 'marked'
```

Comment:
Without this change, attempts to reference `marked.Renderer` failed with marked as undefined. This was caught by our register user integration test (which shows the terms of service by calling `renderAndSanitizeMarkdown`.